index.html

```diff
@@ -211,5 +211,6 @@ <h2>Get started</h2>
                 </ul>
             </div>
         </footer>
+        <script src="/nav-bar.js"></script>
```

Comment:
- [ ] **OPTIONAL**
To keep your codebase clean and organized, I highly suggest two ways for that! The first is to make one (main) javascript file for the project (ex: app.js) while you keep the logic code from the DOM manipulation apart within that file. The second is to create a `js` directory, then put inside of it as many js files as you need for your project. 👍